examples/vite-demo-vanilla-bundle/src/examples/example19.ts

```diff
@@ -91,6 +91,14 @@ export default class Example19 {
           : String.fromCharCode('A'.charCodeAt(0) + (Math.floor(i / 26)) - 1) + String.fromCharCode('A'.charCodeAt(0) + (i % 26)),
         field: String(i),
         minWidth: 60,
+        exportWithFormatter: true,
+        formatter: (row: number, cell: number, value: any) => {
```

Comment:
I don't think we have to add argument types anymore, TypeScript should be smart enough to return at least the first 2 arguments type `number`